plugin.json

```
@@ -4,7 +4,7 @@
   "Name": "TenorGIF",
   "Description": "Search Tenor for amazing GIFs!",
   "Author": "Garulf",
-  "Version": "1.1.2",
+  "Version": "1.2.0",
```

```suggestion
  "Version": "2.0.0",
```

This should be considered a major version as its a breaking change.